apollo-federation/src/query_graph/graph_path.rs

```diff
@@ -774,6 +775,7 @@ enum UnadvanceableReason {
     NoMatchingTransition,
     UnreachableType,
     IgnoredIndirectPath,
+    UnsatisfiableOverrideCondition,
```

Comment:
This looks like it's unused, would it be part of composition validation? if so i'd favour adding it when necessary...